zk-token-sdk/src/encryption/auth_encryption.rs

```diff
@@ -104,14 +118,19 @@ impl AeKey {
         Ok(result.to_vec())
     }
 
+    /// Generates a random authenticated encryption key.
+    ///
+    /// This function is randomized. It internally samples a scalar element using `OsRng`.
     pub fn new_rand() -> Self {
         AuthenticatedEncryption::keygen()
     }
 
+    /// Encrypts an amount under the authenticated encryption key.
     pub fn encrypt(&self, amount: u64) -> AeCiphertext {
         AuthenticatedEncryption::encrypt(self, amount)
     }
 
+    /// Recovers an encrypted amount from an authenticated encryption ciphertext.
     pub fn decrypt(&self, ct: &AeCiphertext) -> Option<u64> {
```

Comment:
There are a few places in the crate that use `ct` for ciphertext, and a few that use `ctxt`. Imo, both should be spelled out. Again, not a change for this PR.

Reply:
Yes, you are right. I made a PR to rename variable names: https://github.com/solana-labs/solana/pull/31828